src/main/modules/nunjucks/index.ts

```diff
@@ -3,8 +3,9 @@ import * as express from 'express';
 import * as nunjucks from 'nunjucks';
 
 export class Nunjucks {
-  constructor(public developmentMode: boolean) {
+  constructor(public developmentMode: boolean, public i18next) {
```

Comment:
can be strongly typed
import * as i18next from 'i18next';
i18next: i18next.i18n